project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.0.0
+sbt.version = 0.13.16
```

Comment:
Just for the record though, I understand this is the only way to cross-build. You need to be running 0.13.16.